Blog Post Showcasing Data Reuse (#74)
dandi/dandi.github.io: _posts/data_reuse_000458.md

> # Unraveling Anesthesia's Effect on Brain Activity

Reviewer: probably needs the metadata header like

```suggestion
---
layout: post
title: Darta Reuse Showcase
author: Catalystneuro team
---

# Unraveling Anesthesia's Effect on Brain Activity
```

_Data Reuse Highlight: This blog entry is part of an ongoing series that showcases examples of data reuse and reanalysis using publicly available data on DANDI._

> Have you ever wondered what happens to your brain when you're under anesthesia? What neural changes coincide with that dramatic loss of consciousness? In this post, we'll explore how two different research teams investigated this fascinating question using the same data from DANDI Archive's Dandiset #000458.

Reviewer: in general might help to review/provide feedback if was 1 sentence a line.

> Have you ever wondered what happens to your brain when you're under anesthesia? What neural changes coincide with that dramatic loss of consciousness? In this post, we'll explore how two different research teams investigated this fascinating question using the same data from DANDI Archive's Dandiset #000458.

Reviewer: ```suggestion
Have you ever wondered what happens to your brain when you're under anesthesia? What neural changes coincide with that dramatic loss of consciousness? In this post, we'll explore how two different research teams investigated this fascinating question using the same data from [DANDI Archive's Dandiset #000458](https://dandiarchive.org/dandiset/000458).
```

So people could go to it right away.

## The Original Study: Peering into the Anesthetized Brain

> In June 2023, Dr. Leslie Claar and colleagues from the Allen Institute published their groundbreaking findings in the journal _eLife_ in the article, “Cortico-thalamo-cortical interactions modulate electrically evoked EEG responses in mice”. Their approach was comprehensive: they stimulated mouse cortex while simultaneously recording with EEG and neuropixels probes, comparing brain activity during wakefulness and under isoflurane anesthesia.

Reviewer: needs ref/link to the article

In awake mice, Claar's team observed a robust event-related potential (ERP) when they delivered pulsatile electrical stimulation to deep layers of the motor cortex. This ERP coincided with a distinct pattern of neural spiking, both locally in the motor cortex and in distant sensorimotor-related thalamic nuclei. The response followed a three-act play: initial excitation within 25 milliseconds, a period of quiescence for about 125 milliseconds, and finally, a strong rebound excitation before returning to baseline.
When the mice were under anesthesia, the story changed dramatically. The ERP's amplitude and complexity plummeted in response to the same stimulation. While the initial excitation persisted in the motor cortex, it vanished in the thalamic nuclei. The rebound excitation, along with its corresponding ERP component at around 180 milliseconds, was significantly weakened.

Based on these observations, Claar and colleagues proposed that interactions between the cortex and thalamus are responsible for the rebound excitation phase of the stimulus response. This rebound excitation, in turn, drives the second component of the ERP – a crucial signal of wakefulness.

> After their experiments, Claar and her team published their data on DANDI, ensuring that it would remain publicly accessible for any future researchers who wanted to use it.

Reviewer: was it before or after article was published?

it is kinda important since if before article published  -- we can show this as an example of sharing before publication and ripping benefits from people discovering new stuff 'in addition' etc.  If shortly after -- also could be some story to tell ;-)

Author: The Dandiset and publication were developed and released pretty much simultaneously.

Dandiset was created 3/16/23 and last updated 6/16/23.
First version of the publication was released 2/9/23, and final version was published 6/26/23.

Reviewer: ok, so sharing was delayed until a publication and initiated after an initial paper draft.  May be worth adding to the story something along the line of "Dataset preparation was done in concert with paper going through revisions to potentially satisfy curious paper reviewers, and helped to ensure that dataset includes all relevant to the article  data and metadata".

## A New Perspective: Zooming in on Synaptic Potentials

> Just a few months later, in November 2023, Dr. Richard Burman and colleagues at the University of Oxford wrote a paper that included reanalysis of this data. Their findings were published in the journal _Neuron_ with the article, “Active cortical networks promote shunting fast synaptic inhibition _in vivo_.” They also investigated the effects of anesthesia on brain activity, but narrowed their focus to the differential function of a specific receptor subtype: GABA<sub>A</sub>R.

Reviewer: ref/link to paper/doi.

> Burman's team used a technique called _in vivo_ gramicidin perforated patch-clamp recording, combined with optogenetic activation of GABAergic synaptic inputs. They compared the GABA<sub>A</sub> receptor equilibrium potential (E<sub>GABA<sub>A</sub>R</sub>) in mice during wakefulness and under urethane anesthesia.

Reviewer: if it was possible to add references for canonical papers or sites describing those techniques in short (wikipedia?), some lame readers like me could appreciate.

The population spiking data from Dandiset [#000458](https://dandiarchive.org/dandiset/000458) serves as a crucial bridge between these levels, allowing Burman et al. to explicitly connect their work to the broader field of anesthesia research.

However, it's important to note some key differences between the two studies. Claar et al. used isoflurane anesthesia and stimulated layers 5/6 of the cortex, while Burman et al. used urethane anesthesia and recorded from layer 2/3.
Additionally, while Claar et al. found the most dramatic differences between waking and anesthetized states around 180 ms after stimulation, Burman et al. focused their reanalysis on the initial spiking response within 15 ms of stimulation.

These differences highlight the potential for further reanalysis of Dandiset [#000458](https://dandiarchive.org/dandiset/000458).
Future studies could explore more similar experimental conditions, conduct a more thorough treatment of relevant response time periods, or investigate different aspects of anesthesia's effects altogether.

> ## A Call to Action: Engaging with the Open Science Community

Reviewer 2: ```suggestion
## Engaging with the Open Science Community
```

We encourage readers from all backgrounds to take advantage of this open science platform by publishing and reanalyzing data on DANDI.
We are confident that it will be a smooth and rewarding experience, but don't just take our word for it -- listen to what Dr. Leslie Claar and Dr. Paul Brodersen have to say,

> "After organizing our dataset into NWB format, validating and uploading our files was straightforward using a few simple commands from the DANDI Python Client. 
> The tools are well-documented, which made the process of uploading and publishing quite smooth. 
> Within a few months of publishing on DANDI, others were already interested in using our dataset."
> -- Dr. Leslie Claar

> “The DANDI archive is a fantastic resource with great tooling and exhaustive documentation. Its command line interface makes data retrieval and synchronization as simple as possible (but not simpler!).
> The standardized and well-documented formats for electrophysiological and neuroimaging data ensure that anyone can quickly find their way around a new archive without requiring extensive feedback from the authors that originally collected the data.

> > These features make the DANDI archive a treasure trove of useful and re-usable data, unlike the unfortunately still commonplace unstructured data dumps on many other platforms or -- worst of all -- university servers.”

Reviewer: `re-usable` is a typo here.  I see that it is a quote but to simplify our lives, I think it would be ok to just fix it up:

```suggestion
> These features make the DANDI archive a treasure trove of useful and reusable data, unlike the unfortunately still commonplace unstructured data dumps on many other platforms or -- worst of all -- university servers.”
```
